alf/data_structures.py

```diff
@@ -481,3 +481,15 @@ def make_experience(time_step: TimeStep, alg_step: AlgStep, state):
         "priority",
     ],
     default_value=())
+
+# some basic structures used by offline replay buffer
+# reconstruction
+Info = namedtuple("Info", [
+    'rl',
+    "rewards",
+    "repr",
+])
+
+RLInfo = namedtuple("RLInfo", [
```

Comment:
These two namedtuple names are too general

Reply:
This is indeed intended to be a very general and basic structure which is shared among different algorithms.